### DISCUSSION TITLE
求助，怎么才能把学校电断了

### DISCUSSION THREAD

**Discussion by H02681** [share121/inter-knot#3137] [其他]
如题，临近开学，不想上学，希望整点大的延迟开学（玩个抽象别当真
![img-1724069751195245253821dc229d5446df8572c4452aeb0f09cc0361e518bf5cd19e700176a9b](https://github.com/user-attachments/assets/5ec1d224-ebb9-428d-b58c-3a367af5adb2)
）

**Comment by I-ikun** [comment]
b站有硝化甘油的制作教程
